libs/feature/search/src/lib/filter-dropdown/filter-dropdown.component.ts

```diff
@@ -46,10 +35,24 @@ export class FilterDropdownComponent implements OnInit {
 
   constructor(
     private searchFacade: SearchFacade,
-    private searchService: SearchService
+    private searchService: SearchService,
+    private aggregationsService: AggregationsService
   ) {}
 
   ngOnInit() {
+    this.choices$ = this.aggregationsService
```

Comment:
It would be better to aggregate the aggregations into one api call, but I think this could be good for now.
Good call